### ISSUE TITLE
`zero_grad_` for global optimizer

### ISSUE THREAD

**Issue by kingnobro** [yael-vinker/live_sketch#3]
Hi Rinon.

In [this function](https://github.com/yael-vinker/live_sketch/blob/f45492110d57272b8b756d0411e22fa67782362f/painter.py#L489C9-L489C19), there is only `points_delta_optimizer.zero_grad()`. Why there is no `global_delta_optimizer.zero_grad()`. Did I miss something? Thanks.

**Comment by rinongal** [comment]
Seems like a bug 😮
As far as I can see in our history, this exists in all of our experiments as well.

I'll leave it up for now for reproducibility and see about adding a flag to fix it + check what parameter changes are needed to compensate. May take a few weeks until I can get to that though.

**Comment by kingnobro** [comment]
No worry. Optimization without global transformation has already achieved good results.

Without `global_delta_optimizer.zero_grad()`, the gradients are not cleared so the parameter update is more significant, I guess. So I think you could increase the global learning rate.
